src/app/modules/shared-components/components/section-paragrah/section-paragraph.component.ts

```diff
@@ -21,6 +21,8 @@ export class SectionParagraphComponent implements OnInit {
   @Input() hasBorder = false;
   @Input() data: any;
   @Input() remainOrigin = true;
+  @Input() isCollapsedByDefault = false;
```

Comment:
With the current behavior, changing the "collapsible" wouldn't work, and there is some strange boolean logic going on so I've added this variable to avoid breaking it.

I could change the logic and make the state of the paragraph depend on an "isCollapsed" if it's better ?

Reply:
See the remark I've just done.

Reply:
fixed